Makefile

```diff
@@ -13,12 +13,20 @@ PLATFORMS ?= linux_amd64 linux_arm64
 -include build/makelib/common.mk
 
 # ====================================================================================
-# Setup Images
+# Setup Output
 
-# even though this repo doesn't build images (note the no-op img.build target below),
-# some of the init is needed for the cross build container, e.g. setting BUILD_REGISTRY
--include build/makelib/image.mk
-img.build:
+# S3_BUCKET ?= crossplane.releases
+-include build/makelib/output.mk
+
+# ====================================================================================
+# Setup Helm
+
+HELM_BASE_URL = https://charts.crossplane.io
+HELM_S3_BUCKET = crossplane.charts
+HELM_CHARTS_DIR=$(ROOT_DIR)/charts
+HELM_CHARTS = oam-core-runtime
```

Comment:
There are some inconsistencies with naming here. The helm chart is named `oam-core-runtime`. The binary and Docker image are `oam-runtime`. The repo is `oam-kubernetes-runtime`. Any thoughts on a consistent name?

Reply:
I have no particular opinion on the specific names but I agree that it's better to be the same. How about they are all called oam-core-runtime ? Where is the image name located?

Reply:
I think they should all be called `oam-kubernetes-runtime` align with the repo.

Reply:
@ryanzhang-oss image name is set here https://github.com/crossplane/oam-kubernetes-runtime/pull/90/files#diff-b67911656ef5d18c4ae36cb6741b7965R58

Does `oam-kubernetes-runtime` sound good to you?

Reply:
sure